tools/cargo_hack_checksum

```diff
@@ -0,0 +1,3 @@
+CIDL_SHA256_DARWIN="ee00750378126c7e14402a45c34f95ed1ba4be2ae505b0c0020bb39b5b3467a4"
```

Comment:
What does `CIDL` stand for here?

Reply:
I figured it was a content identifier to some degree :), but I was going off https://github.com/search?q=repo%3Aoxidecomputer%2Fomicron+cidl&type=code.

Reply:
Ah!